### PR TITLE
Correcting Query Tool Css

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

removing turquoise boxes on cui alert
ensuring link color is correct
correcting th background color

## Why?

NAC-1117

## This PR has:

- [X] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [ ] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [ ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

**Comment by ccaprio97** [comment]
@dborgen2  will need to go over if this is the right way to handle changing things in the wwwroot

File: query-tool/src/Piipan.QueryTool.Client/Components/MatchResults.razor.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿a:link {
+    color: #005ea2;
```
Comment (dborgen2) [inline query-tool/src/Piipan.QueryTool.Client/Components/MatchResults.razor.css:2]: The JIRA ticket made it seem like all links were the wrong color. Does this need to be applied more widely than just MatchResults and QueryResults components?

Comment (dborgen2) [reply query-tool/src/Piipan.QueryTool.Client/Components/MatchResults.razor.css:2]: Also, making a note here to ask Corey about visited link colors. I mentioned this offline, but wanted a record of it

File: query-tool/src/Piipan.QueryTool/wwwroot/sass/_uswds-theme-custom-styles.scss [hunk 2]
```diff
@@ -20,9 +20,28 @@ i.e.
...
+        border-left: none;
+    }
+}
+
+a:link {
+    color: #005EA2;    
+}
+
+a:link.a:hover {
+    color: red;
```
Comment (dborgen2) [inline query-tool/src/Piipan.QueryTool/wwwroot/sass/_uswds-theme-custom-styles.scss:35]: Is this right? red links?

File: query-tool/src/Piipan.QueryTool/wwwroot/sass/_uswds-theme-custom-styles.scss [hunk 3]
```diff
@@ -20,9 +20,28 @@ i.e.
...
-  @include u-bg('cyan-5');
+    @include u-bg('cyan-5');
+
+    .usa-alert {
+        border-left: none;
+    }
+}
+
+a:link {
+    color: #005EA2;
```
Comment (dborgen2) [inline query-tool/src/Piipan.QueryTool/wwwroot/sass/_uswds-theme-custom-styles.scss:31]: It looks to me like you are only styling unvisited links here, and below you're styling visited links inside alerts. Do we need to style visited links that are not inside alerts? Do we still need component-specific styling for the results table?

Comment (ccaprio97) [reply query-tool/src/Piipan.QueryTool/wwwroot/sass/_uswds-theme-custom-styles.scss:31]: so i think i checked and some of them didnt need to be fixed and i think all the places that werent done automatically

File: query-tool/src/Piipan.QueryTool/wwwroot/sass/_uswds-theme-custom-styles.scss (hunk 2 above)
Comment (ccaprio97) [reply query-tool/src/Piipan.QueryTool/wwwroot/sass/_uswds-theme-custom-styles.scss:35]: sorry that was for a test

**Review by bgfoshay — APPROVED** [review]
👍

**Pull request merged**